frontend/src/containers/MetricsStep/SouceControlConfiguration/index.tsx

```diff
@@ -83,6 +85,8 @@ export const SourceControlConfiguration = () => {
     totalSourceControlNumber !== 0 &&
     loadingCompletedNumber === totalSourceControlNumber;
 
+  const isShowRemoveButton = pipelineTools.type === PIPELINE_TOOL_OTHER_OPTION ? totalSourceControlNumber > 1 : true;
```

Comment:
Do we need the logic that only support remove when totalSourceControlNumber > 1?
How about always allow to remove for source control